src/pages/People.jsx

```diff
@@ -0,0 +1,29 @@
+import React, {useEffect} from 'react';
+import {Person} from '../components/Person/Person';
+import {useDispatch, useSelector} from 'react-redux';
+import {dataLoad} from '../redux/dataLoad.js';
+
+//import people from "../people.json";
+import './page.css';
+
+export const People = () => {
+  const people = useSelector(state => state.data);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    function load() {
+      dispatch(dataLoad);
+    }
+    load();
+  }, [dispatch]);
+
+  console.log(people);
```

Comment:
Remove console logs. 
You should pass clean and production-ready code to PR. Otherwise, add `WIP:` prefix to the PR name

WIP = Work In Progress